tests/providers/qubole/operators/test_qubole.py

```diff
@@ -180,3 +192,16 @@ def test_parameter_pool_passed(self):
         test_pool = 'test_pool'
         op = QuboleOperator(task_id=TASK_ID, pool=test_pool)
         assert op.pool == test_pool
+
+    @mock.patch('qds_sdk.commands.Command.get_results', new=get_result_mock)
+    def test_get_results_with_headers(self):
+        dag = DAG(DAG_ID, start_date=DEFAULT_DATE)
+
+        task = QuboleOperator(task_id=TASK_ID, command_type='prestocmd', dag=dag)
+
+        with mock.patch.object(task, 'xcom_pull', return_value='test_command_id'):
+            with mock.patch('qds_sdk.resource.Resource.find', return_value=PrestoCommand):
+                results = open(task.get_results(ti=task, include_headers=True), 'r').read()
+                assert results == RESULTS_WITH_HEADER
+                results = open(task.get_results(ti=task, include_headers=False), 'r').read()
+                assert results == RESULTS_WITH_NO_HEADER
```

Comment:
The two asserts actually test two different test cases so lets you seperate it to 2 tests